Framework/Kernel/inc/MantidKernel/TimeROI.h

```diff
@@ -47,6 +47,7 @@ class MANTID_KERNEL_DLL TimeROI {
   bool operator==(const TimeROI &other) const;
   /// print the ROI boundaries to a string
   std::string debugStrPrint(const std::size_t type = 0) const;
+  void debugPrint(const std::size_t type = 0) const;
```

Comment:
It looks like this doesn't have an implementation since there is no change to `TimeROI.cpp`

Reply:
Good catch! I thought I had gotten rid of that method.
Obliterating...